tests/manage/pv_services/test_verify_thick_pvc_utilization.py

```diff
@@ -0,0 +1,130 @@
+import logging
+import pytest
+from os import path
+
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier2,
+    polarion_id,
+    skipif_ocs_version,
+)
+from ocs_ci.helpers.helpers import default_thick_storage_class, fetch_used_size
+from ocs_ci.ocs import constants
+
+
+log = logging.getLogger(__name__)
+
+
+@skipif_ocs_version("<4.8")
+class TestVerifyRbdThickPvcUtilization(ManageTest):
+    """
+    Tests to verify storage utilization of RBD thick provisioned PVC
+    """
+
+    @pytest.fixture(autouse=True)
+    def setup(self, project_factory):
+        """
+        Create project for the test
+
+        Returns:
+            OCP: An OCP instance of project
+
+        """
+        self.proj_obj = project_factory()
+
+    @tier2
+    @polarion_id("OCS-2537")
+    def test_verify_rbd_thick_pvc_utilization(
+        self,
+        pvc_factory,
+        pod_factory,
+    ):
+        """
+        Test to verify the storage utilization of RBD thick provisioned PVC
+        """
+        pvc_size = 15
+        replica_size = 3
+        file1 = "fio_file1"
+        file2 = "fio_file2"
+
+        size_before_pvc = fetch_used_size(constants.DEFAULT_BLOCKPOOL)
```

Comment:
Using this constant will lead to failure if the test is run on an external mode cluster.
We can use helper function default_ceph_block_pool() to get the default pool name.

Reply:
Made the necessary change.

Reply:
The constant is being used in few other places in the test. Please check those as well.

Reply:
Thanks Sidhant.
Removed  the usage of ```constants.DEFAULT_BLOCKPOOL``` in the test.